src/Lexer.js

```diff
@@ -46,6 +46,7 @@ function Token(text, data, position) {
 var tokenRegex = new RegExp(
     "([ \r\n\t]+)|(" +                                // whitespace
     "---?" +                                          // special combinations
+    "|[\u00C0-\u00FF]" +
```

Comment:
The `\]-\u2027` range should already include this, since `\]` is `\u005D`. If you look at `[!-\[\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]` as a whole, that's almost any BMP codepoint except for the C0 control sequences `\u0000-\u001F`, the space `\u0020`, the backslash `\u005C`, the line separator `\u2028` and the paragraph separator `\u2029` which you might consider whitespace, the surrogate characters of UTF-16 `\uD800-\uDFFF` which are only matched if they form a complete surrogate pair, and the private use area `\uE000-\uF8FF`. So to make it short, this change should not be neccessary, in my opinion.